src/assisted_installer_controller/operator_handler.go

```diff
@@ -256,17 +260,17 @@ func (handler ClusterServiceVersionHandler) deleteFailedSubscriptionInstallPlans
 	return nil
 }
 
-func (handler ClusterServiceVersionHandler) GetStatus() (models.OperatorStatus, string, error) {
+func (handler ClusterServiceVersionHandler) GetStatus() (models.OperatorStatus, string, string, error) {
 	csvName, err := handler.kc.GetCSVFromSubscription(handler.operator.Namespace, handler.operator.SubscriptionName)
 	if err != nil {
-		return "", "", err
+		return "", "", "", err
 	}
 	csv, err := handler.kc.GetCSV(handler.operator.Namespace, csvName)
 	if err != nil {
-		return "", "", err
+		return "", "", "", err
 	}
 	operatorStatus := utils.CsvStatusToOperatorStatus(string(csv.Status.Phase))
-	return operatorStatus, csv.Status.Message, nil
+	return operatorStatus, csv.Status.Message, csv.Spec.Version.String(), nil
```

Comment:
Maybe we should create struct for it? I mean 3 values to return not too much?

Reply:
I agree, can we do it after that PR?

Reply:
yep, just lets not forgot it.

Reply:
Version can't be nil?

Reply:
> yep, just lets not forgot it.

I'll open a task for it



> Version can't be nil?

Nope